wps-cloudformation-template.yaml

```diff
@@ -12,6 +12,9 @@ Parameters:
   s3jobKey:
     Type: String
     Default: 'jobs/'
+  batchLogGroupName:
```

Comment:
Does this need to be an input to the stack?  Inputs should probably be things we want to change per environment.  Adding this as a parameter will require us to modify jenkins to allow it to be specified to update an existing stack.

Reply:
Will update to use /aws/batch/<stack_name> as the log group - no param required